packages/perseus/src/widgets/matrix/matrix.test.ts

```diff
@@ -91,4 +97,58 @@ describe("matrix widget", () => {
         // Assert
         expect(score).toHaveBeenAnsweredIncorrectly();
     });
+
+    describe("interactive: full vs answerless", () => {
+        beforeAll(() => {
+            registerAllWidgetsForTesting();
+        });
+
+        let userEvent: UserEvent;
+        beforeEach(() => {
+            userEvent = userEventLib.setup({
+                advanceTimers: jest.advanceTimersByTime,
+            });
+
+            jest.spyOn(Dependencies, "getDependencies").mockReturnValue(
+                testDependencies,
+            );
+        });
```

Comment:
I just noticed this is available at the top level describe. I can take this chunk out of the nested describe, right?